src/main/java/me/moodcat/database/entities/Room.java

```diff
@@ -45,28 +51,53 @@
     private Integer id;
 
     /**
+     * The name of the room.
+     */
+    @Column(name = "roomName")
+    private String roomName;
+
+    @Embedded
+    @JsonIgnore
+    private VAVector valenceArousal;
+
+    /**
      * The current song of the room.
      */
     @ManyToOne
     @JoinColumn(name = "currentSong")
-    private Song song;
+    private Song currentSong;
```

Comment:
This will break the frontend song retrieval as on https://github.com/MoodCat/MoodCat.me-Frontend/blob/master/app/scripts/controllers/main.js#L73